internal/testutility/utility.go

```diff
@@ -105,7 +105,7 @@ func AssertMatchFixtureText(t *testing.T, path string, actual string) {
 // automatically skipped unless running in a CI environment
 func AcceptanceTests(t *testing.T, reason string) {
 	t.Helper()
-	if os.Getenv("Acceptance_Tests") == "" {
+	if os.Getenv("ACCEPTANCE_TESTS") == "" {
```

Comment:
I think we should probably actually check for a value e.g. `1`:

```suggestion
	if os.Getenv("ACCEPTANCE_TESTS") == "1" {
```

Reply:
I think this is meant to be checking that it's not set, though [`os.LookupEnv`](https://pkg.go.dev/os#LookupEnv) seems to be the more correct way to do it

Reply:
Right good point, have updated my suggestion - `os.LookupEnv` is not inherently the correct way, because it checks if its set at all, not if it's both set _and_ set to an empty value.

I personally think at least in this case `os.LookupEnv` isn't what we want because that typically makes it more work to disable i.e. I have to completely remove `ACCEPTANCE_TESTS` from my commands rather than just setting it to an alternative value

Reply:
I set it to `true` to match the `TEST_NO_DIFF` env we currently have.

Reply:
actually that's a good point 😅 probably best to call it `TEST_ACCEPTANCE` for consistency...